AgentAssistAngular/src/app/services/root.service.ts

```diff
@@ -1044,4 +963,21 @@ export class RootService {
     }
   }
 
+  updateDefaultFeedbackData(){
+    for(let key in this.defaultFeedbackSettings){
+      this.defaultFeedbackSettings[key].negativeKeyWords = {};
+      Object.keys(this.defaultSuggestionObj[key])?.forEach(language => {
+        let langArray = [];
+        this.defaultSuggestionObj[key][language]?.forEach((element, index) => {
+          let obj : any = {
+            isEnabled : index <= 4 ? true : false,
```

Comment:
@saradadh-Kore here 3 nested loops are defined, try to reduce.

Reply:
resolved, you can start review @sathishra-Kore